src/util/request.js

```diff
@@ -68,6 +68,7 @@ const entityPath = (suffix) =>
 export const apiPaths = {
   // Backend generates session tokens that are URL-safe.
   session: (token) => `/v1/sessions/${token}`,
+  currentSession: () => '/v1/sessions/current',
```

Comment:
I usually only add an `apiPaths` function when there's a variable/argument. No harm in adding a constant function, but I also think it'd be OK to hard-code it where it's used.